includes/definitions/discovery/scalance.yaml

```diff
@@ -0,0 +1,156 @@
+mib: SN-MSPS-SCX200-MIB
+modules:
+    os:
+        sysDescr_regex: '/Siemens, SIMATIC NET, (?<hardware>SCALANCE X.+, HW: .+\d), FW:.+/'
```

Comment:
You should probably split this to the part after `,` is part of `feature` instead.
I would do like this; `/SCALANCE (?<hardware>[^,]+), (?<feature>[^,]+)/`